examples/gno.land/r/demo/art/gnoface/gnoface.gno

```diff
@@ -0,0 +1,125 @@
+package gnoface
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/rand"
+	"gno.land/p/demo/ufmt"
+)
+
+func Render(path string) string {
+	seed := std.GetHeight()
+
+	path = strings.TrimSpace(path)
+	if path != "" {
+		s, err := strconv.Atoi(path)
+		if err != nil {
+			panic(err)
+		}
+		seed = int64(s)
+	}
+
+	output := ufmt.Sprintf("Gnoface #%d\n", seed)
+	output += Draw(seed)
+	return output
+}
+
+func Draw(seed int64) string {
+	var (
+		hairs = []string{
+			"     s",
+			"  .......",
+			"   s s s",
+			"   /\\ /\\",
+			"  |||||||",
+		}
+		headtop = []string{
+			" /-------\\",
+			" /~~~~~~~\\",
+			" /|||||||\\",
+			" ////////\\",
+			" |||||||||",
+			" /\\\\\\\\\\\\\\\\",
+		}
+		headspace = []string{
+			" |       |",
+		}
+		eyebrow = []string{
+			"~",
+			"*",
+			"_",
+			".",
+		}
+		ear = []string{
+			"o",
+			" ",
+			"D",
+			"O",
+			"<",
+			">",
+			".",
+			"|",
+			")",
+			"(",
+		}
+		eyesmiddle = []string{
+			"| o   o |",
+			"| o   _ |",
+			"| _   o |",
+			"| .   . |",
+			"| O   O |",
+			"| v   v |",
+			"| X   X |",
+			"| x   X |",
+			"| X   D |",
+			"| ~   ~ |",
+		}
+		nose = []string{
+			" |   o   |",
+			" |   O   |",
+			" |   V   |",
+			" |   L   |",
+			" |   C   |",
+			" |   ~   |",
+			" |  . .  |",
+			" |   .   |",
+		}
+		mouth = []string{
+			" |  __/  |",
+			" |  \\_/  |",
+			" |   .   |",
+			" |  ___  |",
+			" |  ~~~  |",
+			" |  ===  |",
+			" |  <=>  |",
+		}
+		headbottom = []string{
+			" \\-------/",
+			" \\~~~~~~~/",
+			" \\_______/",
+		}
+	)
+
+	r := rand.FromSeed(seed)
+
+	return "```\n" +
+		pick(r, hairs) + "\n" +
+		pick(r, headtop) + "\n" +
+		pick(r, headspace) + "\n" +
+		" | " + pick(r, eyebrow) + "   " + pick(r, eyebrow) + " |\n" +
+		pick(r, ear) + pick(r, eyesmiddle) + pick(r, ear) + "\n" +
+		pick(r, headspace) + "\n" +
+		pick(r, nose) + "\n" +
+		pick(r, headspace) + "\n" +
+		pick(r, mouth) + "\n" +
+		pick(r, headspace) + "\n" +
+		pick(r, headbottom) + "\n```\n"
```

Comment:
Assuming that the 3 back ticks are here for the markdown rendering, I think they should be added in `Render`, not in `Draw`.